vm/src/builtins/weakproxy.rs

```diff
@@ -88,6 +88,7 @@ impl PyWeakProxy {
         self.try_upgrade(vm)?.repr(vm)
     }
 
+    #[pymethod(magic)]
```

Comment:
does weakproxy has contains method?

Reply:
[Related Test Link](https://github.com/RustPython/RustPython/blob/9fd31b8f2c2f8ab3bfa1182eac496f3b3b102973/Lib/test/test_weakref.py#L246)
In the test, there is a checking process that proxy has \_\_contain\_\_().

Reply:
how do you create actual weakproxy object having `__contains__` attribute? I failed

```
>>> class A(list): pass
... 
>>> a = A()
>>> w = weakref.ref(a)
>>> w.__contains__
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'weakref' object has no attribute '__contains__'
>>> 
```

Reply:
Actually, `weakproxy` has `__contain__` as a `slot-wrapper` method which is added by sequence protocol in cpython. But in rustpython, there is no such mechanism about it yet. So, I think this can be a **stopgap**.

Reply:
I confimed that `__contains__` is not included in `python 3.10.5`.
```
>>> dir(weakref)
['CallableProxyType', 'KeyedRef', 'ProxyType', 'ProxyTypes', 'ReferenceType', 
'WeakKeyDictionary', 'WeakMethod', 'WeakSet', 'WeakValueDictionary', 
'_IterationGuard', '__all__', '__builtins__', '__cached__', '__doc__', 
'__file__', '__loader__', '__name__', '__package__', '__spec__', 
'_collections_abc', '_remove_dead_weakref', 'finalize', 
'getweakrefcount', 'getweakrefs', 'itertools', 'proxy', 'ref', 'sys']
```

The `__contains__` method is only for referred instance (aka. actual instance), so if we tested this method, it should be used for them, not for weakref(or weakproxy) instance.

For example in `RustPython`,
```
>>>>> L = [1,2]
>>>>> r = weakref.ref(L)
>>>>> ref_r = r()
>>>>> type(ref_r)
<class 'list'>
>>>>> r.__contains__
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'weakref' object has no attribute '__contains__'
>>>>> ref_r.__contains__
<bound method list.__contains__ of [1, 2]>
```

If anything wrong to me, plz tell me.

Reply:
Weren't we talking about `weakproxy`, not `weakref`?
For checking `__contains__` in `weakproxy`
```python
>>> dir(weakref.ProxyType)
['__abs__', '__add__', '__and__', '__bool__', '__bytes__', '__class__', '__contains__', '__delattr__', '__delitem__', '__dir__', '__divmod__', '__doc__', '__eq__', '__float__', '__floordiv__', '__format__', '__ge__', '__getattribute__', '__getitem__', '__gt__', '__hash__', '__iadd__', '__iand__', '__ifloordiv__', '__ilshift__', '__imatmul__', '__imod__', '__imul__', '__index__', '__init__', '__init_subclass__', '__int__', '__invert__', '__ior__', '__ipow__', '__irshift__', '__isub__', '__iter__', '__itruediv__', '__ixor__', '__le__', '__len__', '__lshift__', '__lt__', '__matmul__', '__mod__', '__mul__', '__ne__', '__neg__', '__new__', '__next__', '__or__', '__pos__', '__pow__', '__radd__', '__rand__', '__rdivmod__', '__reduce__', '__reduce_ex__', '__repr__', '__reversed__', '__rfloordiv__', '__rlshift__', '__rmatmul__', '__rmod__', '__rmul__', '__ror__', '__rpow__', '__rrshift__', '__rshift__', '__rsub__', '__rtruediv__', '__rxor__', '__setattr__', '__setitem__', '__sizeof__', '__str__', '__sub__', '__subclasshook__', '__truediv__', '__xor__']
```

Reply:
@Snowapril Yeah, I know `__contains__` is in `weakproxy`, not `weakref`.
The reason I refer on it is @youknowone used `__contains__` with `weakref` above the execute result.
So I just want to say it is **correct result** that error message is displayed when using `__contains__` on `weakref`.